sidekiq/README.md

```diff
@@ -56,9 +56,15 @@ No additional installation is needed on your server.
        prefix: "sidekiq."
        mappings:
          - match: "sidekiq.sidekiq.*"
```

Comment:
Does it need to be "sidekiq.sidekiq.(.*)" to capture $1?

Reply:
It doesn't need to be regex (there are a few examples in airflow https://github.com/DataDog/integrations-core/blob/master/airflow/README.md#step-2-connect-airflow-to-dogstatsd-included-in-the-datadog-agent-by-using-airflow-statsd-feature-to-collect-metrics), but adding it does make the capturing more robust and allows capturing "dots" in the title